SQL/tgstation_schema.sql

```diff
@@ -527,6 +527,14 @@ CREATE TABLE `achievements` (
 	PRIMARY KEY (`ckey`,`achievement_key`)
 ) ENGINE=InnoDB;
 
+DROP TABLE IF EXISTS `achievement_metadata`;
+CREATE TABLE `achievement_metadata` (
+	`achievement_key` VARCHAR(32) NOT NULL,
+	`achievement_version` SMALLINT UNSIGNED NOT NULL DEFAULT 0,
+	`achievement_type` VARCHAR(50) NULL DEFAULT NULL,

```

Comment:
This could be an enum but it's not really important, maybe not worth bother if the table has already been made.

Reply:
Table is empty so there's no rush, i'll switch this

Reply:
@optimumtact redo DB stuff please